nextcord/embeds.py

```diff
@@ -373,15 +385,27 @@ def set_footer(
             The footer text.
         icon_url: :class:`str`
             The URL of the footer icon. Only HTTP(S) is supported.
+        icon_file: Optional[:class:`File`]
+            A file to use for the image.
+
+            .. versionadded:: 2.1
         """
+        from . import InvalidArgument
```

Comment:
Can this not be imported at the top?

Reply:
Currently not, but I can try find a work around